src/react/hooks/useLazyQuery.ts

```diff
@@ -128,12 +191,10 @@ export function useLazyQuery<
             ...executeOptions,
             fetchPolicy: executeOptions.fetchPolicy || initialFetchPolicy,
           }
-        : {
-            fetchPolicy: initialFetchPolicy,
-          };
+        : { fetchPolicy: initialFetchPolicy };
 
```

Comment:
No need to change these lines.
```suggestion
        : {
            fetchPolicy: initialFetchPolicy,
          };
```

Reply:
Ah ya meant to fix this after removing my first attempt at this fix. Thanks!